frontstage/common/session.py

```diff
@@ -18,49 +17,36 @@ def from_session_key(cls, session_key):
         else:
             encoded_jwt_token = redis.get(session_key)
         session = cls(session_key, encoded_jwt_token)
-        session.persisted = True
+
         return session
 
     @classmethod
     def from_party_id(cls, party_id):
-        """Create a new and unpersisted session object from a party_id, this will encode a JWT
-        and set an expiry time, it will default the unread message count to 0"""
+        """Create a new session object from a party_id, this will encode a JWT
```

Comment:
Perhaps historically from_party_id was used by something else which needed it not to be persisted and set, in this instance it now is only used on sign_in.py, which instantly persists and set afterwards, basically we don't need to do that anymore

Reply:
Yes, I think this was a workaround to a bug in the count not being handled correctly under some scenarios. Fortunately removing the feature removed the need to know if the session object has been persisted in Redis or not.